CogNative/main.py

```diff
@@ -0,0 +1,59 @@
+import os
+import wave
+
+from models.RTVC.RTVC import RTVC
+
+# INITIALIZE RTVC
+v = RTVC("models/RTVC/saved_models/default")
+
+# SET INPUT AUDIO FILE PATH
+file_path = input("Enter input audio file path:\n")
+assert os.path.exists(file_path), "ERROR: Path not found."
+
+# ENTER TEXT
+text = input("Enter text for voice clone:\n")
+assert text[-1] == ".", "ERROR: Punctuation missing."
+
+# OUTPUT FILE PATH
+output_path = input("Enter output audio path:\n")
```

Comment:
Need to check for correct output path. If a folder is input here, it will run the synthesis and then crash. Specify that a wav file needs to be included in the end of the path here.

Reply:
I added better checks for the input and output file paths.